packages/ui/src/data/filters/utils.ts

```diff
@@ -0,0 +1,272 @@
+import qs from 'query-string';
```

Comment:
it seems like a bit "utils". I think it needs to be split into meaningful and specialized parts.

Reply:
It could be, but every fonction here is to play with filter/query params and lots of functions are related to each other.